### PR TITLE
fix: fixing readme, launcher.py, etc

### PR DESCRIPTION
**Pull request by decave27**

### CODE REVIEW

File: launcher.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python launcher.py
```
Comment (minibox24) [inline launcher.sh:3]: doesn't work on linux, do i need to put this in?

**Review by minibox24 — CHANGES_REQUESTED** [review]
refactor launcher.py -> src/launcher.py

File: launcher.sh (hunk 1 above)
Comment (decave27) [reply launcher.sh:3]: You won't have to put it in.

**Comment by decave27** [comment]
> 
> 
> refactor launcher.py -> src/launcher.py

✅

**Review by minibox24 — CHANGES_REQUESTED** [review]
remove `launcher.sh`

**Review by minibox24 — APPROVED** [review]

**Pull request merged**